sdcm/logcollector.py

```diff
@@ -1014,6 +1014,8 @@ class BaseSCTLogCollector(LogCollector):
         FileLog(name='result_gradual_increase.log'),
         FileLog(name='partition_range_scan_diff_*.log',
                 search_locally=True),
+        FileLog(name='junit.xml',
```

Comment:
Why is it in the base class?
Should be in the K8S log collector.

Reply:
it should come with the SCT logs, it has nothing to do with k8s